### PR TITLE
Update 2019-03-13-build-rest-api-with-aspnet-web-api.md

### PR DESCRIPTION
**Pull request by nickgamb**
Removed old Okta steps and added steps using CLI. Updated Authorization Server setup.

This blog post has the following features:

- [X] A GitHub Repository with a polished README
- [X] A GitHub Repository under the github.com/oktadeveloper account
- [X] A title that's approved by Dev Advocacy
- [X] A URL that's approved by Dev Advocacy
- [X] The content has been ran through Grammarly (https://www.grammarly.com/)
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md [hunk 1]
```diff
@@ -261,26 +261,28 @@ Here's how it works:
 
 (If this is all new to you, you can [learn more about OAuth here](https://developer.okta.com/blog/2017/06/21/what-the-heck-is-oauth).)
 
-You probably don't want to build an OAuth 2.0 authorization server yourself, so for this tutorial, you will sign up for a [forever-free Okta developer account](https://developer.okta.com/signup/) (or sign in to `{yourOktaDomain}` if you already have an account).
+You probably don't want to build an OAuth 2.0 authorization server yourself, so for this tutorial, you will sign up for a forever-free Okta developer account; or sign in to `{yourOktaDomain}` if you already have an account). Install the [Okta CLI](https://cli.okta.com/) and run `okta register` to sign up for a new account. If you already have an account, run `okta login`.
```
Comment (mraible) [inline _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md:264]: You should be able to replace this whole section with:

```
{% include setup/cli.md type="service" %}
```

If there are improvements you need to make when type == service, please feel free to update `cli.md`.

**Review by mraible — CHANGES_REQUESTED** [review]

File: _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md (hunk 1 above)
Comment (nickgamb) [reply _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md]: Updated

File: _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md [hunk 2]
```diff
@@ -261,26 +261,28 @@ Here's how it works:
...
-* Select **Service** and click **Next**
-* Enter `Postman` for the **Name** and click **Done**
+* Run `okta apps create`
+* Enter `Postman` for the **Name** and press enter
+* Choose `4` for the app type to create a `Service` app
+* Choose `5` to create the app using a framework of `Other`
+* Select the `default` Authorization Server and press enter  
 
-Note the **Client ID** and the **Client Secret**. This is the username and password that our client application (Postman in our case) need to send to the authorization server to request an access token. You can make note these credentials now or you can come back and get them later when you need them.
+Run `cat .okta.env` (or type `.okta.env` on Windows) to see the issuer and credentials for your app. Note the **Client ID** and **Client Secret**. This is the username and password that our client application (Postman in our case) need to send to the authorization server to request an access token. You can make note these credentials now or you can come back and get them later when you need them.
```
Comment (mraible) [inline _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md:276]: Ideally, all these steps can be removed and the include can print out all of this. I'll take a look and see if this is possible w/o too much trouble.

Comment (nickgamb) [reply _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md]: Oh yeah the include does cover a lot. heh. Sorry my first time using. Should be good now.

File: _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md [hunk 3]
```diff
@@ -261,26 +261,20 @@ Here's how it works:
...
+The **OKTA_OAUTH2_CLIENT_ID** and **OKTA_OAUTH2_CLIENT_SECRET** is the username and password that our client application (Postman in our case) will need to send to the authorization server to request an access token. You can make note these credentials now or you can come back and get them later when you need them.
 
 ## Create a Custom Scope
 
 Now you need to set up the authorization server:
 
-* Go to **API** > **Authorization Servers**
-* Click the link for the **default** authorization server
-* Select the **Scopes** tab
+* Log into the Okta Admin Console
```
Comment (mraible) [inline _source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md:274]:
```suggestion
* Log into the Okta Admin Console (use `okta login` to get your org's URL)
```

**Review by mraible — APPROVED** [review]
I made one small suggestions, but otherwise, this looks good!

**Pull request merged**